lighthouse-core/report/html/renderer/util.js

```diff
@@ -66,6 +66,7 @@ class Util {
     if (!clone.configSettings.locale) {
       clone.configSettings.locale = 'en';
     }
+    // If LHR is older than v7, it uses emulatedFormFactor instead of formFactor. https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
```

Comment:
```suggestion
    // If LHR is older than v7, it uses emulatedFormFactor instead of formFactor.
    // https://github.com/GoogleChrome/lighthouse/blob/master/docs/emulation.md#changes-made-in-v7
```